API/projectClass/contrastsClass.m

```diff
@@ -139,6 +141,10 @@
                 end
             end
 
+            if ~strcmpi(modelType, modelTypes.StandardLayers.value) && any(contrastRepeatLayers-1)
```

Comment:
I think it might be better if this warning was shown when the user sets the `repeatLayers` value then it can be reset back to 1.

Reply:
Hmm, the problem is that it doesn't know the model type when adding/setting the contrast. Do we want to change the methods such that the model type must always be supplied?